src/test/scala/com/cognite/sdk/scala/common/ReadBehaviours.scala

```diff
@@ -133,6 +133,48 @@ trait ReadBehaviours extends Matchers { this: FlatSpec =>
     }
   }
 
+  private def fetchTestItemsWithRequiredExternalIds[R <: WithRequiredExternalId with WithCreatedTime](readable: Readable[R, Id]): List[R] = {
+    // only use rows older than 10 minutes, to exclude items created by concurrently running tests which might be deleted quickly
+    val minAge = Instant.now().minus(10, ChronoUnit.MINUTES)
+    readable
+      .list()
+      .filter(_.createdTime.isBefore(minAge))
+      .take(2)
+      .compile
+      .toList
+  }
+
+  def readableWithRetrieveByRequiredExternalId[R <: WithRequiredExternalId with WithCreatedTime, W](
```

Comment:
@hakontro Noticed there is not "retrieve by external id" method for labels in API reference. This is only used for relationships. Error handling for the same not existing ids is different than the others for relationships. It does not return ids in missing or duplicated so I excluded that logic. I am actually not happy to copy the code but creating a generic readableWithRetrieveExternalId was tricky because of small differences. I would appreciate any comments if we have a way to do it.